drivers/mil_blueview_driver/src/blueview_ros_driver.cpp

```diff
@@ -167,9 +167,9 @@ void BlueViewRosDriver::loop(const ros::TimerEvent &)
 
 int main(int argc, char **argv)
 {
+  ros::init(argc, argv, "blue_view_driver");
```

Comment:
Lol stickler comment, but I feel like we've been referencing blueview as one word. Consistency is always nice.